jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/MultiPartTest.java

```diff
@@ -58,7 +58,7 @@ public void testParseNoPartsInTwoChunks()
 
         assertEquals(0, listener.events.size());
 
-        parser.parse(Content.Chunk.from(BufferUtil.EMPTY_BUFFER, true));
+        parser.parse(Content.Chunk.EOF);
```

Comment:
My understanding of this PR is that you didn't need to change this, as `from` would have returned EOF anyway.    Just checking that is true?

Reply:
If `Chunk.from` would correctly choose from `EOF`, `EMPTY`, you'd be right.